_haikus/recommendation_systems.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Recommendation Systems
+author: SPrio
+---
+
+This systems recommned<br>
+similar things to the users<br>
+based on their preferences<br>
```

Comment:
If you remove `their` from this line, is will have the correct number of syllables.